src/lib/db/clients/mysql.js

```diff
@@ -321,6 +322,34 @@ export async function updateValues(conn, updates) {
   return results
 }
 
+export async function deleteRows(conn, updates) {
+  const deleteCommands = updates.map(update => {
+    return {
+      query: `DELETE FROM ${wrapIdentifier(update.table)} WHERE ${wrapIdentifier(update.pkColumn)} = ?`,
```

Comment:
Can we use knex here like you did for postgres?

Reply:
Absolutely. I just saw that it wasn't included in this connector, so I thought there might be a reason for that. I'll update it soon.